spec/unit/pluginHandler/windows.spec.js

```diff
@@ -441,40 +451,47 @@ describe('windows project handler', function () {
             });
         });
 
-        describe('of <framework> elements', function () {
+       describe('of <framework> elements', function () {
             // This could be separated into individual specs, but that results in a lot of copying and deleting the
             // project files, which is not needed.
             it('should remove from correct project files when conditions specified', function () {
+                var curDir;
+                curDir = __dirname;
+                process.chdir(path.join(curDir, '..', 'fixtures', 'testProj'));
+
                 var frameworks = copyArray(valid_frameworks);
 
                 frameworks.forEach(function(framework) {
                     PluginHandler.getInstaller('framework')(framework, dummyPluginInfo, dummyProject);
                 });
 
-                var path = 'ItemGroup/Reference';
```

Comment:
A variable named differently from "path" has to be used here. This is because a variable named "path" is already declared at the top of the file. See line 23: var path = require('path');. In addition, path.join() is used in this code, so overwriting the "path" variable will cause us to lose our reference to the path module.